src/ui/routes/initialize/configure/+page.svelte

```diff
@@ -3,51 +3,127 @@
 // SPDX-FileCopyrightText: 2021-Present The Zarf Authors
  -->
 <script lang="ts">
-	import AccordionGroup from '../../../lib/components/accordion-group.svelte';
+	import AccordionGroup from '$lib/components/accordion-group.svelte';
 
-	import Icon from '$lib/components/icon.svelte';
-	import PackageDetails from '$lib/components/package-details-card.svelte';
-	import PackageComponent from '$lib/components/package-component-accordion.svelte';
+	import { SectionHeader, ComponentAccordion as PackageComponent } from '$lib/components/package';
 	import { pkgStore } from '$lib/store';
-	import { Button, Typography } from '@ui';
+	import { Button, Chip, Typography } from '@ui';
+	let showRaw: boolean = false;
```

Comment:
Can we use implicit typing here as we do throughout the rest of the codebase? `let showRaw = false;`